app/main/routes.py

```diff
@@ -28,6 +28,21 @@ def index():
     return render_template("index.html")
 
 
+@bp.route("/logout", methods=["GET"])
+@access_token_login_required
+def logout():
```

Comment:
called this view logout even though the naming of the view we are redirected to after is signed_out, since our login view is called login but I guess I should probably change both logout and login to be sign_out and sign_in?

Reply:
Sounds good, we should go with sign in & sign out. The phrasing is slightly more formal and welcoming than "login."